From 0054f6aac9b19cb2ce4672f21779488363188f56 Mon Sep 17 00:00:00 2001
From: Denis Barbier <bouzim@gmail.com>
Date: Wed, 11 May 2016 16:28:23 +0200
Subject: [PATCH] Allows building with libc++ on OSX

---
 src/common/context.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/common/context.hpp b/src/common/context.hpp
index 4bd294d..b882268 100644
--- src/common/context.hpp
+++ src/common/context.hpp
@@ -32,7 +32,7 @@
 #include <vector>
 #include <fstream>
 
-#if (__cplusplus > 201103L) || defined(HAVE_CPP11) || defined(_MSC_VER)
+#if (__cplusplus > 201103L) || defined(HAVE_CPP11) || defined(_MSC_VER) || defined(_LIBCPP_VERSION)
   #include <unordered_map>
   #define UM_NS std
 #else
